src/components/textfield/textfield.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TextfieldLabel } from './../typography/typography';
+
+const StyledLabel = styled(TextfieldLabel)`
+  display: block;
+  margin-bottom: 4px;
+
+  &.error {
+    font-weight: 700;
+    color: #ff0d35;
+  }
+`;
+
+interface Textfield {
+  errorMessage: string;
+  label: string;
+}
+
+const Textfield = ({ label, errorMessage }) => {
```

Comment:
This is not quite what I meant with "introduce an interface". I was hoping for a interface for the properties of this component, like so:

```suggestion
interface TextfieldProps {
  errorMessage: string;
  label: string;
}

const Textfield = ({ label, errorMessage }: TextfieldProps) => {
```

This way we can benefit from proper types inside the component.
Before:
![image](https://user-images.githubusercontent.com/7970458/142614526-2432fa45-47a8-4e30-a448-2e5e15197a26.png)

After:
![image](https://user-images.githubusercontent.com/7970458/142614554-e5f44810-a6c8-4416-bbf2-97b1dfd10cd0.png)